npm/webpack-dev-server/src/loader.ts

```diff
@@ -15,8 +17,8 @@ const makeImport = (file: Cypress.Cypress['spec'], filename: string, chunkName:
   const magicComments = chunkName ? `/* webpackChunkName: "${chunkName}" */` : ''
 
   return `"${filename}": {
-    shouldLoad: () => document.location.pathname.includes(${JSON.stringify(file.relative)}),
-    load: () => import(${JSON.stringify(path.resolve(projectRoot, file.relative))} ${magicComments}),
+    shouldLoad: () => document.location.pathname.includes(${JSON.stringify(file.absolute)}),
+    load: () => import(${JSON.stringify(path.resolve(projectRoot, file.absolute))} ${magicComments}),
```

Comment:
`path.resolve` does not need the `projectRoot`, as the file path is absolute.

Reply:
Fixed.